jolpica/formula_one/models/round.py

```diff
@@ -4,11 +4,22 @@
 
 from django.db import models
 
+from jolpica.formula_one.models.session import SessionType
+
 if TYPE_CHECKING:
     from . import Session, SessionEntry
     from .managed_views import DriverChampionship, TeamChampionship
 
 
+class RoundType(models.TextChoices):
+    """The type of round"""
+
+    CONVENTIONAL = "C"
+    SPRINT_V1 = "S1"
+    SPRINT_V2 = "S2"
+    SPRINT_V3 = "S3"
+
+
```

Comment:
This should be using SessionType.

Round is made of sessions, of which some of those sessions are sprints.

We don't want to put a round type here, as we'll just end up with a long list of all possible types and combinations of weekends. 
We should pick this up implicitly from the sessions found inside a round, as this will make it much easier to adapt to future changes

Reply:
I modified it to only work based on the sessions in a round. That leaves the issue that this creates many additional queries.

Reply:
I'd say don't worry too much about queries right now. 
If we need to optimise them in the future we can

Reply:
I've increased the acceptable number of queries for this endpoint only in the tests.